thorlcr/thorutil/thbuf.cpp

```diff
@@ -2549,6 +2543,9 @@ class CSharedFullSpillingWriteAhead : public CInterfaceOf<ISharedRowStreamReader
         outputStream->flush();
         totalInputRowsRead.fetch_add(newRowsWritten);
         tempFileOwner->noteSize(iFileIO->getStatistic(StSizeDiskWrite));
+        CRuntimeStatisticCollection currentFileStats(inactiveStats.queryMapping());
```

Comment:
is it worth having some functions like the following in jstats::

```
template <class INTERFACE>
void updateStatsDelta(CRuntimeStatisticCollection & fullStats, CRuntimeStatisticCollection & deltaStats, INTERFACE * source)
{
    CRuntimeStatisticCollection curStats(deltaStats.queryMapping());
    mergeStats(curStats, source);
    fullStats.updateDelta(deltaStats, curStats);
}

```
then this would be

```
updateStatsDelta(inactiveStats, previousFileStats, iFileIO);
```